api/src/index.js

```diff
@@ -1,15 +1,35 @@
 const path = require('path');
+const Router = require('./routes');
+const postgres = require('./postgres');
 
 const envPath = path.resolve(__dirname, '.env.' + process.env.NODE_ENV);
 require('dotenv').config({
   path: envPath,
 });
-
+const passport = require('passport');
 const express = require('express');
+const cookieSession = require('cookie-session');
 const app = express();
-const Router = require('./routes');
+const isLoggedIn = require('./loggedIn.js');
```

Comment:
If we don't have routes in this file, there's no reason to have isLoggedIn imported here